config/profile_stub.config

```
@@ -20,6 +21,7 @@ params{
   // path to reference files
   ref_json = "test/stub-refs.json"
   ref_rootdir = 'test/references'
+  
```

Turns out you do need these... I have a thought about how to make them update automatically, but I'm not sure it will work, so better to just do it here for now at least
```suggestion
  
  // cell type references directories
  celltype_ref_dir = "${params.ref_rootdir}/celltype"
  // output from save_singler_refs() process
  singler_references_dir = "${params.celltype_ref_dir}/singler_references"
  // output from train_singler_models() process, and input to classify_singler()
  singler_models_dir = "${params.celltype_ref_dir}/singler_models"
  // output from generating cell assign reference matrices, and input to classify_cellassign()
  cellassign_ref_dir = "${params.celltype_ref_dir}/cellassign_references"
```